crates/rpc/rpc/src/otterscan.rs

```diff
@@ -68,7 +71,7 @@ where
 #[async_trait]
 impl<Eth> OtterscanServer for OtterscanApi<Eth>
 where
-    Eth: EthApiServer<RpcTransaction<Eth::NetworkTypes>, RpcBlock<Eth::NetworkTypes>>
+    Eth: FullEthApiServer
```

Comment:
this is increasing the set of behaviour available to `Eth`, this is the opposite of what we want. we want to reduce the set of behaviour available to `Eth`, to that which it really needs. I think when you're done addressing this review, it will be enough with trait bound `Eth: EthTransactions`.
```suggestion
    Eth: EthTransactions<RpcTransaction<Eth::NetworkTypes>, RpcBlock<Eth::NetworkTypes>>
```

Reply:
@emhane Thanks again, I add `EthTransactions` to otterscan instead of the full EthApiServer, this is really less limit.